### DISCUSSION TITLE
My Gravatar not replace with new profile photo after update it

### DISCUSSION THREAD

**Discussion by budioktaviyan** [community/community#43084] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hi team!

I had recently update my profile photo in Gravatar. Apparently, the profile photo from my gravatar still uses the old profile picture

Here is my Gravatar profile picture which was recently updated
<img width="282" alt="Screen Shot 2023-01-02 at 08 57 19" src="https://user-images.githubusercontent.com/2031493/210190550-3a16d290-5730-49e0-a286-028365477041.png">

This is what it looks like in my Github profile
<img width="761" alt="Screen Shot 2023-01-02 at 08 58 07" src="https://user-images.githubusercontent.com/2031493/210190560-caa0f1fa-69a4-4259-a1f1-b2be215b1d77.png">

**Comment by radar-blip** [comment]
It's possible the selected answer here might help: #32845

**Comment by myConsciousness** [comment]
Hi @budioktaviyan ,

Profile images and other changes take time to take effect. You can wait up to 30 minutes for the changes to take effect, but have you waited longer than that?

**Reply by budioktaviyan** [reply]
I have waited longer than that for months, and the problem is still there
Now, I still use a manual upload to change my profile photo instead of using a Gravatar

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @budioktaviyan! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Reply by budioktaviyan** [reply]
Hi @queenofcorgis thanks for the reply and really appreciate your helping me fix the issue
Can you share with me the new link to discuss this issue? since the problem still sits here

Here is what it looks like in my GitHub profile right now
<img width="811" alt="Screen Shot 2023-02-24 at 21 33 55" src="https://user-images.githubusercontent.com/2031493/221206320-ecaff8ff-12cc-490c-8771-02d501ed1e5f.png">

This is my Gravatar profile pic
<img width="366" alt="Screen Shot 2023-02-24 at 21 34 15" src="https://user-images.githubusercontent.com/2031493/221206494-7a411441-06a1-483a-a410-5d1f7893cf8d.png">

**Comment by Kaparrs** [comment]
Maybe try removing the profile picture you've uploaded on GitHub? The uploaded image might be overriding the Gravatar one.

---

![image](https://user-images.githubusercontent.com/71441748/221221321-89645009-6d54-4670-982f-f50ad3e5b4cb.png)

**Reply by budioktaviyan** [reply]
I have tried, but it replaces the very old pictures (not sure why). this is the problem I guess

**Reply by Kaparrs** [reply]
That's weird

**Comment by xyz327** [comment]
hi, did you resolved this problem?  
i updated my gravatar profile. and deleted my github profile avatar, but github avatar not fetch my gravatar pic. now is a  randomly generated pic
how can i let github use my gravater?

when i sign up on github, my gravatar is not create
maybe github set avatar use gravatar  just  in sign up?

**Reply by budioktaviyan** [reply]
The problem didn't solved until now
